src/main/scripts/src/app/core/rest/organization.service.ts

```diff
@@ -19,40 +19,81 @@
 
 @Injectable()
 export class OrganizationService {
 
-  constructor(private httpClient: HttpClient) {
+  constructor(private httpClient: HttpClient,
+    private workspaceService: WorkspaceService) {
   }
 
   public getOrganizations(): Observable<Organization[]> {
-    return this.httpClient.get<Organization[]>(OrganizationService.apiPrefix());
+    return this.httpClient.get<Organization[]>(this.apiPrefix());
   }
 
   public getOrganization(code: string): Observable<Organization> {
-    return this.httpClient.get<Organization>(OrganizationService.apiPrefix(code));
+    return this.httpClient.get<Organization>(this.apiPrefix(code));
   }
 
   public deleteOrganization(code: string): Observable<HttpResponse<object>> {
-    return this.httpClient.delete(OrganizationService.apiPrefix(code), {observe: 'response'});
+    return this.httpClient.delete(this.apiPrefix(code), {observe: 'response'});
   }
 
   public createOrganization(organization: Organization): Observable<HttpResponse<object>> {
-    return this.httpClient.post(OrganizationService.apiPrefix(), organization, {observe: 'response'});
+    return this.httpClient.post(this.apiPrefix(), organization, {observe: 'response'});
   }
 
   public editOrganization(code: string, organization: Organization): Observable<HttpResponse<object>> {
-    return this.httpClient.put(OrganizationService.apiPrefix(code), organization, {observe: 'response'});
+    return this.httpClient.put(this.apiPrefix(code), organization, {observe: 'response'});
   }
 
-  private static apiPrefix(code?: string): string {
+  public getPermissions(): Observable<Permissions> {
+    return this.httpClient.get<Permissions>(`${this.actualApiPrefix()}/permissions`)
+      .catch(OrganizationService.handleGlobalError);
+  }
+
+  public updateUserPermission(userPermissions: Permission): Observable<Permission> {
+    return this.httpClient.put<Permission>(`${this.actualApiPrefix()}/permissions/users`, userPermissions)
+      .catch(OrganizationService.handleGlobalError);
+  }
+
+  public updateGroupPermission(userPermissions: Permission): Observable<Permission> {
+    return this.httpClient.put<Permission>(`${this.actualApiPrefix()}/permissions/groups`, userPermissions)
+      .catch(OrganizationService.handleGlobalError);
+  }
+
+  public removeUserPermission(user: string): Observable<HttpResponse<any>> {
+    return this.httpClient.delete(`${this.actualApiPrefix()}/permissions/users/${user}`, {observe: 'response'})
+      .catch(OrganizationService.handleGlobalError);
+  }
+
+  public removeGroupPermission(group: string): Observable<HttpResponse<any>> {
+    return this.httpClient.delete(`${this.actualApiPrefix()}/permissions/groups/${group}`, {observe: 'response'})
+      .catch(OrganizationService.handleGlobalError);
+  }
+
+  private actualApiPrefix(): string {
+    let orgCode = this.workspaceService.currentlySelectedOrganizationCode;
+    return `/${API_URL}/rest/organizations/${orgCode}`;
```

Comment:
You are mixing two different styles here. Already existing methods take `organizationCode` as a parameter while the new ones rely on `WorkspaceService`. I would suggest unifying their behavior and require `code` parameter in every method.